dialect/sql/builder.go

```diff
@@ -2951,7 +2951,7 @@ func (*WithBuilder) view() {}
 // only to query rows-limited edges in pagination.
 type WindowBuilder struct {
 	Builder
-	fn        string // e.g. ROW_NUMBER(), RANK().
+	selector  func(*Builder)
```

Comment:
```suggestion
	fn  func(*Builder) // e.g. ROW_NUMBER(), RANK().
```
It's not a selector anymore.